content/articles/creating-dummy-data-in-python-using-faker-package/index.md

```diff
@@ -0,0 +1,99 @@
+
+For testing and operational reasons, dummy data is used. Your code can be tested against a variety of different types of stimuli.
+### Prerequisites
+1. Before proceeding with this tutorial, you should know the python programming language.
+2. Have a python IDE or follow this [documentation](https://www.python.org/) to install.
+
+> Setup Faker with the pip command:
+
+```Python
+Pip install Faker
+```
+
+### Listed below are methods and types of dummy data
+#### 1. Faker generators can be created and initialized
+You can generate and initialize fake generators using `Faker()`. Using the Faker generator, you'll be able to generate any data you desire.
+```Python
+from faker import Faker
+ourFake = Faker()
+```
+#### 2. Createing random text 
+To create random text, we use `text()`. The method below created a paragraph.
+```Python
+ourFake.text()
+```
+Output:
+```bash
+About eight four they wife.
+Force finally very summer. Occur, you shake through physical reveal. Require player yeah magazine thank bank traditional.
+```
+For addresses we use `address()` and for names we use `name()` methods.
+#### 3. Creating same dummy data
+You may wish to use the same collection of data again. Seeding the generator is a viable option. The following code generates dummy data using the seed() method:
+```Python
+Faker.seed(111)
+print(ourFake.text())
+```
+Output:
+```bash
+Management massive pay college cover instead. Consumer's leg start research her.
+Sound finish set draw notice imagine that. Blue between least democratic down week wait. Reduce inside me.
+```
+#### 4. Creating unique dummy data
+You can utilize the generator's.unique property to ensure that the dummy data created is unique.
+```python
+# generates 10 unique texts 
+texts = [ourFake.unique.name() for b in range(10)]
+```
+#### 5. Command-line usage of Faker package
+Command-line invocation of the faker package allows you to directly type in the command prompt code.
+#### 6. Providers
+For the most part, these features are bundled into service providers.
+1. Standard Providers- You can find more information [here](https://faker.readthedocs.io/en/stable/providers.html).
+2. Community Providers- [This](https://faker.readthedocs.io/en/stable/communityproviders.html) site has a wealth of data on locally developed service providers. 
+#### Creating localized dummy data
+The Faker Generator may generate localized false data if a place is provided as an input. Moreover, it can be translated into a wide range of languages. Otherwise, all locales must use the python list data type. 
+
+The following are some Hindi names we can come up with:
+```Python
+ourFake = Faker('hi_IN')
+for b in range(10):
+    print(ourFake.name())
+```
+Output:
+```bash
+हेगडे, देन्यल
+प्रबोध खान
+रायन कृष्णन
+लाल, नरेन्द्र
+काले, फ़रहान
+प्रभाकर भंडारी
+कुलकर्णी, तृष्णा
+सरस्वती बालासुब्रमणियम
+गौतम दुआ
+विजय कृष्णा
+```

```

Comment:
I'm not sure if Section website supports other languages' alphabets. I would suggest you to replace with some other language that contains English alphabets.